contracts/erc721/InvoiceUnpaidNFT.sol

```diff
@@ -125,8 +125,8 @@ contract PaymentObligation is Initializable, UserMintableERC721 {
     _mandatoryFields.push(INVOICE_DUE_DATE);
 
     UserMintableERC721.initialize(
-      "Centrifuge Payment Obligations",
-      "CENT_PAY_OB",
+      "Centrifuge Unpaid Invoices",
+      "CENT_UNPAID_INVOICE",
```

Comment:
change to `CENT_INVOICE_UNPAID` 
the idea of leading with the object, followed by the status, is that if we have different `INVOICE` tokens then it is easier to sort/order etc if the leading part is the object, followed by the status